src/reducers/editors.ts

```diff
@@ -5,9 +5,10 @@ import createReducer from "../utils/createReducer";
 const initialState = Object.values(editors).reduce((accum, val) => {
   // eslint-disable-next-line no-param-reassign
   accum.push({
-    name: val,
+    name: val.name,
     enabled: true, // Enabled by default for now
-    img: imgPathMap[val]
+    img: imgPathMap[val.name],
```

Comment:
If we use editor classes directly, the base editor class should have an `id` or `key` method that creates a unique slug from `instance.name`.